blog-preview-card/styles.css

```
@@ -0,0 +1,107 @@
+body {
+  background-color: hsl(47, 88%, 63%);
+  font-family: "Figtree";
+  font-weight: 500;
+  margin: 0;
+}
+
+body p {
+  font-size: 18px;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 390px;
+  background-color: hsl(0, 0%, 100%);
+  border: 1px solid hsl(0, 0%, 7%);
+  border-radius: 20px;
+  box-shadow: 8px 8px 0px -3px hsl(0, 0%, 7%);
+  padding: 24px;
+  align-items: left;
+  line-height: 150%;
+}
+
+.card img {
+  border-radius: 20px;
+  width: 100%;
+  height: auto;
+  display: block;
+}
+
+.card a {
+  font-weight: 800;
+  font-size: 24px;
+  text-decoration: none;
+}
+
+.card a:hover {
+  color: hsl(47, 88%, 63%);
+}
+
+.card:hover {
+    box-shadow: 18px 18px 0px -3px hsl(0, 0%, 7%);
+}
```

Just a note for the future:

If you use SASS / SCSS, you can nest everything related to `.card` class:

```suggestion
.card {
    img {
        border-radius: 20px;
        width: 100%;
        height: auto;
        display: block;
    }
  
    a {
        font-weight: 800;
        font-size: 24px;
        text-decoration: none;
    }

    a:hover {
        color: hsl(47, 88%, 63%);
    }

    :hover {
        box-shadow: 18px 18px 0px -3px hsl(0, 0%, 7%);
    }
}
```